xtask/src/ci.rs

```diff
@@ -40,9 +40,9 @@ pub fn ci(args: CiJob, metadata: CargoMetadata) -> cross::Result<()> {
             // Set labels
             let mut labels = vec![];
 
-            let image_title = match target.triplet.as_ref() {
-                "cross" => target.triplet.to_string(),
-                _ => format!("cross (for {})", target.triplet),
+            let image_title = match target.name.as_ref() {
+                "cross" => target.name.to_string(),
+                _ => format!("cross (for {})", target.name),
             };
             labels.push(format!("org.opencontainers.image.title={image_title}"));
             labels.push(format!(
```

Comment:
This should probably have a label for the platform it runs on as well, not just the target. Probably should be:

```rust
"cross (for {target}) on {platform}"
```

(Unless I"m reading this wrong, in which case correct me).

Reply:
Likewise, we should probably add a few more image [fields](https://github.com/opencontainers/image-spec/blob/main/annotations.md):

- `vendor`: `cross-rs/cross`
- `version`: we already have this info.
- `source`: `https://github.com/cross-rs/cross`
- `created`: build date, can be done via the `chrono` library.

The date can be done with something like:

```rust
use chrono::{Datelike, Utc};

pub fn get_current_date() -> Date {
    let utc = Utc::now();
    let date = utc.date();

    format!("{}-{}-{}", date.year(), date.month(), date.day())
}
```

Reply:
I'm going to leave `"cross (for {target}) on {platform}"` as a todo, it's a bit tricky in how to implement it, and it could be misleading